library/bignum_mod_raw.h

```diff
@@ -33,6 +33,54 @@
 
 #include "bignum_mod.h"
 
+/**
+ * \brief   Perform a safe conditional copy of MPI which doesn't reveal whether
+ *          the condition was true or not.
+ *
+ * \param[OUT] X        The address of the first MPI. This must be initialized.
```

Comment:
Should we include something about the size of `X` and `A` like we do in the import/export functions, where we say "The size is determined by \p m"?